src/SerialDataCollector.cpp

```diff
@@ -746,9 +748,7 @@ namespace splash
     {
         this->fileStatus = FST_CREATING;
 
-        // appends the mpiPosition to the filename (e.g. myfile_0_1_0.h5)
```

Comment:
what can be stored in `full_filename`? Please add some examples as before (line removed above) just to make it a bit more transparent